tests/env/start-swift-ceph.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
via the travis file you can control the vm which is used.

if it works on one setup but not on the other there is maybe a diff in the vm config.

please compare the yaml files